### PR TITLE
Use AudioWorklet instead of ScriptProcessor

### PR DESCRIPTION
**Pull request by beaufortfrancois**
This PR promotes `AudioWorklet` instead of `ScriptProcessor` in the "Access the raw data from the microphone" section from "Recording Audio from the User" blog posts.

Live preview: https://deploy-preview-9175--web-dev-staging.netlify.app/media-recording-audio/#access-the-raw-data-from-the-microphone

### CODE REVIEW

File: src/site/content/en/blog/media-recording-audio/index.md [hunk 1]
```diff
@@ -130,32 +130,45 @@ Web Audio API is a simple API that takes input sources and connects those
...
 ultimately to a speaker so that the user can hear it.
 
-One of the nodes that you can connect is a `ScriptProcessorNode`. This node will
-emit an `onaudioprocess` event every time the audio buffer is filled and you
-need to process it. At this point, you could save the data into your own buffer
-for later use.
+One of the nodes that you can connect is an `AudioWorkletNode`. This node gives
+you the low-level capability for custom audio processing. The actual audio
+processing happens in the `process()` callback method in the `AudioWorkletProcessor`.
+This function is called to feed inputs and parameters and fetch outputs.
```
Comment (AaronForinton) [inline src/site/content/en/blog/media-recording-audio/index.md:136]:
```suggestion
Call this function to feed inputs and parameters and fetch outputs.
```

[Active voice](https://developers.google.com/style/voice)

**Review by AaronForinton — CHANGES_REQUESTED** [review]
Thanks for this one. Do you know who could double-check the JS? @beaufortfrancois

**Comment by beaufortfrancois** [comment]
@hoch Can you have a quick look at this small change?
Context: https://bugs.chromium.org/p/chromium/issues/detail?id=1073292

File: src/site/content/en/blog/media-recording-audio/index.md [hunk 2]
```diff
@@ -130,32 +130,45 @@ Web Audio API is a simple API that takes input sources and connects those
...
-One of the nodes that you can connect is a `ScriptProcessorNode`. This node will
-emit an `onaudioprocess` event every time the audio buffer is filled and you
-need to process it. At this point, you could save the data into your own buffer
-for later use.
+One of the nodes that you can connect is an `AudioWorkletNode`. This node gives
+you the low-level capability for custom audio processing. The actual audio
+processing happens in the `process()` callback method in the `AudioWorkletProcessor`.
+Call this function to feed inputs and parameters and fetch outputs.
+
+Checkout [Enter Audio Worklet](https://developer.chrome.com/blog/audio-worklet/)
```
Comment (mjwilson-google) [inline src/site/content/en/blog/media-recording-audio/index.md:138]: I think this be "Check out" instead.

Comment (AaronForinton) [reply src/site/content/en/blog/media-recording-audio/index.md]: Fixed this! Are you happy to sign off on the little bit of JS? @mjwilson-google

**Comment by hoch** [comment]
Sorry for the late response, but I'd like to defer my review to @mjwilson-google and @alvinjiooo. :)

**Review by AaronForinton — APPROVED** [review]

File: src/site/content/en/blog/media-recording-audio/index.md (hunk 2 above)
Comment (mjwilson-google) [reply src/site/content/en/blog/media-recording-audio/index.md]: Thanks for the fix!  I didn't check the JavaScript in detail; if you still need a full review please ping me again (may not get to it today).

**Pull request merged**